lib/utils/validate-params.ts

```diff
@@ -74,3 +76,26 @@ export function checkIncludeContentSourceMapsParamIsAllowed(
 
   return includeContentSourceMaps as boolean
 }
+
+export function checkEnableTimelinePreviewIsAllowed(
+  host?: string,
+  timelinePreview?: TimelinePreview,
+) {
+  if (timelinePreview === undefined) {
+    return false
+  }
+
+  const isValidConfig = isValidTimelinePreviewConfig(timelinePreview)
+
+  const isValidHost = host === 'preview.contentful.com'
```

Comment:
We should probably stick to just checking the subdomain. To enable testing with staging and preview

Reply:
Ah, I had a feeling that I missed something important